enos/enos-modules.hcl

```diff
@@ -104,7 +104,7 @@ module "aws_target" {
 }
 
 module "vault" {
-  source = "app.terraform.io/hashicorp-qti/aws-vault/enos"
+  source = "./modules/aws_vault"
```

Comment:
For now, I'm going to just copy the contents of the of the shared terraform provider as opposed to updating to use the copy from the vault repo. There have been many a number of changes to this module that exist in the vault repo, so I'd like to save that for a separate activity.